src/OpenLoco/src/S5/S5.h

```diff
@@ -410,23 +424,36 @@ namespace OpenLoco::S5
         std::vector<std::pair<ObjectHeader, std::vector<uint8_t>>> packedObjects;
     };
 
-    namespace LoadFlags
+    enum class LoadFlags : uint32_t
+    {
+        none = 0U,
+        titleSequence = 1U << 0,
+        twoPlayer = 1U << 1,
+        scenario = 1U << 2,
+    };
+    OPENLOCO_ENABLE_ENUM_OPERATORS(LoadFlags);
+
+    constexpr bool hasLoadFlags(LoadFlags flags, LoadFlags flagsToTest)
```

Comment:
The load and save flags are just for the load and save functions they shouldn't need to be in the header.

Reply:
Ahhh i remember now what my comment was about. Its the hasLoad/SaveFlags functions. Move them into the .cpp file they shouldn't be in the header as they aren't used by anything apart from the one .cpp file.